go-controller/pkg/cluster/cluster.go

```diff
@@ -34,7 +34,14 @@ const (
 	// OvnHostSubnet is the constant string representing the annotation key
 	OvnHostSubnet = "ovn_host_subnet"
 	// OvnClusterRouter is the name of the distributed router
-	OvnClusterRouter = "ovn_cluster_router"
+	OvnClusterRouter              = "ovn_cluster_router"
+	OvnNodeGatewayMode            = "ovn_node_gateway_mode"
+	OvnNodeVlanID                 = "ovn_node_vlan_id"
+	OvnNodeIfaceID                = "ovn_node_iface_id"
+	OvnNodeMacAddress             = "ovn_node_mac_address"
```

Comment:
For vlanid, ifaceid, and macaddress, can we put 'gateway' in the name/annotation too? Since these are specific to the gateway functionality.  So perhaps

`OvnNodeGatewayVlanID` + `ovn_node_gateway_vlan_id`
`OvnNodeGatewayIfaceID` + `ovn_node_gateway_iface_id`
<etc>